scripts/generate-inline-script-hash.js

```diff
@@ -24,14 +28,15 @@ fs.readFile("./www/index.html", "utf8", (error, data) => {
         validResults.push(sha256Script);
       }
     }
-    fs.readFile("./manifest.json", "utf8", (error, data) => {
+
+    fs.readFile("./src/webplugin/" + process.argv[2] + "/manifest.json", "utf8", (error, data) => {
```

Comment:
If using synchronous writing in the callback wouldn't it also be easier to use synchronous reading here? Since async behavior does not appear to be important.